README.md

```diff
@@ -1 +0,0 @@
-# Agroeasy
```

Comment:
Why did you delete the README.md

Reply:
You actually use it to explain the summary of what's happening in the repo most times or to pass information or maybe give instructions. It does not affect the code/files in the repo but good practice for good comprehension of the repo